api/src/main/java/org/openmrs/api/InvalidActivationKeyException.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api;
+
+import org.openmrs.api.context.Context;
+
+/**
+ * Represents often fatal errors that occur when a user cannot be found due to incorrect, expired
+ * token or wrong email and or username.
```

Comment:
These javadocs seem incorrect, i thought the exception is only thrown for invalid or expired activation key